src/OpenTelemetry.Instrumentation.EntityFrameworkCore/OpenTelemetry.Instrumentation.EntityFrameworkCore.csproj

```diff
@@ -7,7 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="0.4.0-beta.2" />
+    <PackageReference Include="OpenTelemetry" Version="0.5.0-beta.2" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
@ejsmith Microsoft.SourceLink.GitHub ref should come from a props or targets file in a parent dir. I see one at the top of the files changed, would you double-check this is needed?

Reply:
@CodeBlanch it is not currently in a Directory.Build file, but I could add it there. I think that makes sense.

Reply:
Move it to the Director.Build.targets file.

Reply:
we have in the props one. You don't need to include

Reply:
@eddynaka I'm not seeing it. Which props file?

Reply:
@ejsmith, my bad, wrong name:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/b57e88de71e7795fe9696d271238e20cb2eaacb3/src/Directory.Build.targets#L4-L9

Reply:
Ok, so my change should be good then. I think adding it to the directory.build file is a good thing since we don't want different projects using different versions.

Reply:
i think u can remove the github section that u added to the .csproj.

Reply:
@ejsmith We're talking about the `<PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0">` block in OpenTelemetry.Instrumentation.EntityFrameworkCore.csproj. Please remove that, it should already be inlcuded via the targets file.

Reply:
@CodeBlanch Ahhhh ok. Sorry about that. Will get it updated.

Reply:
Removed.